benchmark-compare.js

```diff
@@ -107,11 +120,23 @@ if (!choices.length) {
       `Throughput/Mb\n(% of ${base.name})`
     ]
   })
+  const arrayObjChoices = []
   data.forEach(result => {
     const beBold = result.server === 'fastify'
     const { hasRouter = false, version } = info(result.server) || {}
     const getPct = (base, value) => ((value / base * 100).toFixed(2))
 
+    arrayObjChoices.push(
+      {
+        name: result.server,
+        version,
+        hasRouter,
+        requests: result.requests.mean,
+        latency: result.latency.mean,
+        throughput: result.throughput.mean
```

Comment:
This should be BYTE, meanwhile, in the previous code edit it is converted to MB.
Am I wrong?

Reply:
Yes..you are right @Eomm thank you, sorry for missing this one - I'm not sure how I missed it in the second if.